src/Backend/Modules/MediaLibrary/Installer/Uninstaller.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Backend\Modules\MediaLibrary\Installer;
+
+/*
+ * This file is part of Fork CMS.
+ *
+ * For the full copyright and license information, please view the license
+ * file that was distributed with this source code.
+ */
+
+use Backend\Core\Installer\AbstractModuleUninstaller;
+use Backend\Core\Installer\UninstallerInterface;
+
+/**
+ * Uninstaller for the MediaLibrary module
+ */
+class Uninstaller extends AbstractModuleUninstaller implements UninstallerInterface
+{
+    public function uninstall(): void
+    {
+        $this->setModule('MediaLibrary');
+
+        $this->deleteBackendNavigation();
+
+        $this->dropDatabaseTables(['MediaItem', 'MediaGroupMediaItem', 'MediaGroup', 'MediaFolder']);
```

Comment:
Table names are false. They currently look like: `MediaLibraryMediaItem`...
I would rather see a function using the Entity class. MediaItem::class, ... Like in the installer

Reply:
That won't work because the table name doesn't need to match the class name